README.md

```diff
@@ -44,12 +44,37 @@ gulp.task('default', function () {
 				// file is the vinyl file object
 				// cb is the through2 callback
 
-				// return false to continue the stream
+				// default: throws an error
+
+				// return false to continue the stream or true to stop the stream
+			},
+			onNotFound: function (file, cb) {
+				// file is the vinyl file object
+				// cb is the through2 callback
+
+				// default: continues the stream
+
+				// return false to continue the stream or true to stop the stream
 			}
 		}));
 });
 ```
 
+Regex
+
+```js
+    .pipe(plugins.contains({
+      search: /([ \t]*\n){4,}/, // Looks for multiple empty lines
+      regex: true,
```

Comment:
Is a `regex` option really needed? Surely you can just detect whether `search` is a string or regex?